languages/Dockerfile.compile.j2

```diff
@@ -2,6 +2,11 @@ FROM {{builder_image}} as builder
 USER 2222
 # Docker build commands don't resolve environment variables so need this to either be numeric or a build argument
 COPY --chown=2222:2222 algosource /opt/algorithm/
+
+{% if config.src_path is defined %}
```

Comment:
i mean i guess i still have the question as to what the contents of this directory are and why it is added for local compilation.  scanning code i'm not really certain but maybe it's something with trying to avoid installing dependencies on every local build or something, but i would really rather not have this in here even though it can add compile time for local development.  we have a ticket # 2469 for getting dependency caches from the host to work properly and i would rather have it work for both local and cluster builds rather than have an extra parameter that isn't used otherwise

Reply:
I added it to make my life easier for when deploying a required dependency to a third party repository was expensive to resolve (maven, CRAN, etc), so adding this allows you to migrate a local package to the docker image for testing. If you are planning on enabling that through a dedicated host -> container dependency pipeline I can definitely remove it or put it in a separate PR.

Reply:
Renamed the variable to be a bit more articulate